src/components/pages/Categories/List.vue

```diff
@@ -53,13 +58,27 @@ export default {
     errorMessage() {
       return this.$store.state.categories.errorMessage;
     },
+    deleteCategoryName() {
+      return this.$store.state.categories.deleteCategory.name;
+    },
   },
   created() {
     this.$store.dispatch('categories/getAllCategories');
     this.$store.dispatch('categories/initCategory');
     this.$store.dispatch('categories/clearMessage');
   },
   methods: {
+    openModal(categoryId, categoryName) {
+      const categoryData = { id: categoryId, name: categoryName };
+      this.$store.dispatch('categories/confirmDeleteCategory', categoryData);
```

Comment:
[nits]

- `Data` という単語はあまり変数名には用いない方が良いかと思います。
  - 「データ」という言葉の定義が曖昧で、どんな変数にもつけようと思えば付けられてしまい、その基準が分かりにくいため
  - こういった時は変数の命名をするとしたらシンプルに `category` とかで良いかと思います。

- ただ、今回の場合、変数に格納する必要は特にないかなと思います。
  - 一時変数になってしまっているかと思いますが、他の開発メンバーがここのコードをみた時、「変数に代入した後何が行われるのだろう？」という一瞬の迷いが生じてしまうためです。
  - その迷いを引き起こしてしまう可能性があるので、一時変数は削除するように以下のようにするのが良いかなと思います。

```suggestion
      this.$store.dispatch('categories/confirmDeleteCategory', { id: categoryId, name: categoryName });
```

Reply:
86532619617c89de40ee80e7b99ab1f28b93d1cf
上記の通りに修正したのですが、挙動は問題なく動くものの、文字数オーバーを示すESLintエラーが出てしまいました。
送るときだけでも一時変数にした方がいいのか、
ここで受け取るときの引数名をid、nameなど、簡潔な名前にした方がよいのか、
それ以外で、ここからidとnameの両方をactionに渡して起爆する方法があるのか。が疑問です。。